include/blmc_drivers/blmc_joint_module.hpp

```diff
@@ -624,6 +634,26 @@ class BlmcJointModules
         }
     }
 
+    /**
+     * @brief Perform homing for all joints at endstops.
+     *
+     * See BlmcJointModule::homing_at_current_position for description of the arguments.
+     *
+     * @return Final status of the homing procedure (since homing happens at current,position,
+     * procedure always returns success).
+     */
+    HomingReturnCode execute_homing_at_endstop(Vector home_offset_rad)
```

Comment:
To be consistent with the method of the single module, I'd also call this `*_at_current_position` here